spec/bolt/cli_spec.rb

```diff
@@ -747,12 +750,16 @@ def stub_config(config, file_content = nil)
           cli.config.modulepath = [File.join(__FILE__, '../../fixtures/invalid_mods')]
         end
 
-        it "task show displays an error" do
+        it "task show prints a warning but shows other valid tasks" do
           options = {
             mode: 'task',
             action: 'show'
           }
-          expect { cli.execute(options) }.to raise_error(/unexpected token at/)
+          cli.execute(options)
+          json = JSON.parse(output.string)
+          expect(json).to eq([['sample::ok', nil]])
```

Comment:
Where is the `sample::ok` task?  I don't see it in the fixtures.

Reply:
I forgot to `git add` those! Will do.

Reply:
👍